### ISSUE TITLE
Highlighting current ASF path

### ISSUE THREAD

**Issue by kipstafoo** [MegaMek/megamek#393]
When you have multiple ASFs on a map, with crisscrossing paths, it would be great if, during the fire phase, the path of the current ASF could be highlighted in a different color to determine what is actually in its path and what isn't.

**Comment by SirMegaV** [comment]
+1

**Comment by mchapman1970** [comment]
Yes i think this suggestion is outstanding.  Sometimes i run two wings and it is hard to follow their paths.

**Comment by Dylan-M** [comment]
While this is an enhancement, I'm marking it for stable because I feel this is essential for using ASF on the ground map.

**Comment by arlith** [comment]
I should've assigned this to myself; I have a fix for this, but in the course of fixing it, I realized that there's a graphical glitch in how the the flight paths are drawn (bends aren't handled quite correctly, and there's a little triangle on the corners).  I started working on the graphical glitch, which I almost have finished, then my wife went into labor and things got a little hectic.

**Comment by arlith** [comment]
Pretty easy fix.  In `FlyOverSprite.drawOnto`, it now checks to see if the `Entity` that the sprite is for is the currently selected entity, and if it is, it strokes the flight path poly in the "selected" color, `Color.blue`.  This gives a subtle effect, but it's noticeable when you're flipping between entities.

While fixing this issue, I noticed a few other graphical issues which I've reported in  #418.
